eth/peer.go

```diff
@@ -191,41 +190,41 @@ func (p *peer) SendReceiptsRLP(receipts []rlp.RawValue) error {
 // RequestHeaders is a wrapper around the header query functions to fetch a
 // single header. It is used solely by the fetcher.
 func (p *peer) RequestOneHeader(hash common.Hash) error {
-	log.Debug(fmt.Sprintf("%v fetching a single header: %x", p, hash))
+	p.Log().Debug("Fetching a single header", "hash", hash)
```

Comment:
Nitpick: either use "Fetching a <subject>" or "Fetching <subject>". You're mixing the two. Below you're using "Fetching batch of headers", and on this line "Fetching a single header".